keps/prod-readiness/sig-cluster-lifecycle/2915.yaml

```diff
@@ -0,0 +1,3 @@
+kep-number: 2915
+alpha:
+  approver: "@ehashman"
```

Comment:
/assign @ehashman 

looks like we have to include this file if the KEP is "implementable" for 1.23.